app/jobs/state_file/send_reject_resolution_reminder_notification_job.rb

```diff
@@ -14,11 +14,14 @@ def priority
       PRIORITY_LOW
```

Comment:
I think something else we might have to add on line 10 is `send_message(require_verification: false)`; there is some weird bug that Jenny and Anisha showed me earlier where a client can have a mismatch of contact they verified versus the contact that they opted into